web/packages/teleport/src/Bots/Add/GitHubActions/ConfigureBot.tsx

```diff
@@ -181,6 +181,7 @@ export function ConfigureBot({ nextStep, prevStep }: FlowStepProps) {
                     login: e.target.value,
                   })
                 }
+                rule={requireValidSSHUser}
```

Comment:
I wanted to suggest using a `requiredField` validation rule from `shared/components/Validation/rules`, but I noticed it doesn't trim the input. Was it the reason to add your own function?

Reply:
Actually, I just had no clue we had shared helpers for that. I'll switch it in.

Reply:
4fb0d6f4c6